indy_node/server/request_handlers/domain_req_handlers/attribute_handler.py

```diff
@@ -46,24 +46,25 @@ def dynamic_validation(self, request: Request):
                 "for that identity")
 
     def gen_txn_path(self, txn):
+        self._validate_txn_type(txn)
         path = domain.prepare_attr_for_state(txn, path_only=True)
         return path.decode()
 
-    def _update_state_with_single_txn(self, txn, isCommitted=False) -> None:
+    def _update_state_with_single_txn(self, txn, is_committed=True) -> None:
```

Comment:
is_committed needs to be False by default

Reply:
fixed